tests/onnxruntime/test_modeling_ort.py

```diff
@@ -36,6 +42,7 @@ def __init__(self, *args, **kwargs):
         self.LOCAL_MODEL_PATH = "assets/onnx"
         self.ONNX_MODEL_ID = "philschmid/distilbert-onnx"
         self.FAIL_ONNX_MODEL_ID = "sshleifer/tiny-distilbert-base-cased-distilled-squad"
+        self.ONNX_SEQ2SEQ_MODEL_ID = "echarlaix/t5-small-onnx"
```

Comment:
Maybe we could put this model under the [ONNX](https://huggingface.co/OWG) or [optimum](https://huggingface.co/optimum) orgs on the Hub?

Similar for your model @philschmid - WDYT?

Reply:
Great idea !